judge/models/profile.py

```diff
@@ -121,6 +123,10 @@ class Profile(models.Model):
                                       help_text=_('32 character base32-encoded key for TOTP'),
                                       validators=[RegexValidator('^$|^[A-Z2-7]{32}$',
                                                                  _('TOTP key must be empty or base32'))])
+    scratch_codes = EncryptedNullCharField(max_length=255, null=True, blank=True, verbose_name=_('scratch codes'),
+                                           help_text=_('JSON of 16 character base32-encoded codes for scratch codes'),
+                                           validators=[RegexValidator(r'^["\[\],A-Z0-9 ]$',
+                                                                      _('Scratch codes must be empty or base32'))])
```

Comment:
Should probably say something about JSON (e.g `Scratch codes must be empty or a JSON array of 16-character base32 codes` or something along those lines).